readme.md

```diff
@@ -44,11 +50,18 @@ $ docker-compose up -d && docker exec -it nettacker_nettacker_1 /bin/bash
 * `docker-compose` will share your nettacker folder, so you will not lose any data after `docker-compose down`
 * To see the API key in you can run `docker logs nettacker_nettacker_1`.
 * More details and setup without docker https://github.com/OWASP/Nettacker/wiki/Installation
+
+| **Chat Group** | **Google Summer of Code** |
+| -------------- | ------------------------- |
+| [<img src="https://d34u8crftukxnk.cloudfront.net/slackpress/prod/sites/6/2019-01_BrandRefresh_slack-brand-refresh_header-1.png?d=500x500&f=fill" width="100" title="Slack">][Slack]  | [<img src="https://betanews.com/wp-content/uploads/2016/03/vertical-GSoC-logo.jpg" width="100"></img>][GSOC-OWASP] |
```

Comment:
are `d34u8crftukxnk.cloudfront.net` and `betanews.com` trusted sources?

Reply:
dont know ,can we download the images and upload it to our repository ? so that from there we can directly call the image.